### PR TITLE
Add 404 page

### LINKED ISSUE #15: Layout: 404 Page
Should look like [wordpress.org/news](https://wordpress.org/news/asdfasdf).

# Designs
Screen Shot 2022-11-01 at 10 14 52 AM

### PR DESCRIPTION
**Pull request by renintw**
Fixes #15 
Related https://github.com/WordPress/wporg-parent-2021/pull/54

## Screenshots

| Desktop | Tablet | Mobile |
| --- | --- | ---|
| Screen Shot 2022-11-18 at 12 34 18 AM |  Screen Shot 2022-11-18 at 12 39 21 AM | Screen Shot 2022-11-18 at 12 39 10 AM |

### CODE REVIEW

File: source/wp-content/themes/wporg-showcase-2022/templates/404.html [hunk 1]
```diff
@@ -1,8 +1,24 @@
-<!-- wp:template-part {"slug":"header"} /-->
+<!-- wp:template-part {"slug":"header","theme":"wporg-showcase-2022"} /-->
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"blockGap":"0px"}},"className":"entry-content","layout":{"type":"constrained"}} -->
-<main class="wp-block-group entry-content">
-	404
+<!-- wp:group {"tagName":"main","className":"site-content-container"} -->
+<main class="wp-block-group site-content-container">
+	<p class="error404__oops" aria-hidden="true">
+		Oops!
```
Comment (StevenDufresne) [inline source/wp-content/themes/wporg-showcase-2022/templates/404.html:6]: We'll need to localize the strings in this file.

**Comment by StevenDufresne** [comment]
Thanks for the PR!

Ultimately, I do think this should probably be in the parent. I'm curious as to what others think.

The search input doesn't look great.
- We could probably fix this issue in the process: https://github.com/WordPress/wporg-news-2021/issues/395
- The input border width should probably be `1px`
- Since news doesn't really have a search bar (or at least I can't find it), we could use our search bars in this theme as a guide.

I think @ryelle or @iandunn can give you a more accurate review of the styles and what is necessary since they were around for the implementation.

**Comment by StevenDufresne** [comment]
I just saw this issue in the parent theme. 

Link: https://github.com/WordPress/wporg-parent-2021/issues/49

**Comment by StevenDufresne** [comment]
@renintw Can we close this and just track this in the parent?

**Comment by renintw** [comment]
Closed in favor of https://github.com/WordPress/wporg-parent-2021/pull/54

**Pull request closed without merging**